common/focuses/00_test_focuses.txt

```diff
@@ -1,115 +0,0 @@
-﻿#test load
```

Comment:
Is it me or the Light lifestyle looks like it's always selected?
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
Isn't it because you have perks in it?

Reply:
> Isn't it because you have perks in it?

Yes, true.

Reply:
Hm, interesting, vanilla lifestyles are not "always selected" if you have perks in them.
I guess I'll need to investigate this.

Reply:
> Hm, interesting, vanilla lifestyles are not "always selected" if you have perks in them.
> I guess I'll need to investigate this.

Nah, they are actually works the same way, it's just less noticeable.